docs/UserGuide.md

```diff
@@ -206,6 +206,21 @@ Examples:
 * `markOrder 1` marks the order with index 1 from the displayed order list as completed.
 * `listOrder 1` followed by `markOrder 2` selects the 1st person in the address book and marks the 2nd order under the 1st person as completed.
 
+### Filter orders under a Person by order status : `filterOrder`
+
+Filters orders by the specified order status, under a selected person from their displayed order list.
+
+Format: `filterOrder ORDER_STATUS`
+
+* A person's order list must first be displayed before filtering their order list.
+* Filters the selected person's order list for orders that match the `ORDER_STATUS`.
+* The order status **must be a valid status**. e.g. Completed, Pending.
+* `ORDER_STATUS` is case-insensitive. e.g. `completed` is the same as `Completed`.
```

Comment:
Should we add another line that says that ORDER_STATUS can only be completed or pending?

Reply:
Noted on this, I leave open for now in case we add more statuses.